api/src/main/java/com/ford/labs/retroquest/team/validation/PasswordValidator.java

```diff
@@ -36,33 +36,34 @@ public void initialize(PasswordConstraint constraintAnnotation) {
 
     @Override
     public boolean isValid(String value, ConstraintValidatorContext context) {
-        for(PasswordValidityChecker cur:PasswordValidityChecker.values()) {
+        for (PasswordValidityChecker cur : PasswordValidityChecker.values()) {
             cur.validate(scrubForNull(value));
         }
 
         return true;
     }
-    private String scrubForNull(String value){
-        return  value==null?"":value;
+
+    private String scrubForNull(String value) {
```

Comment:
I realize this is just whitespace change on this commit, but there's a stringutils function for precisely this.
`StringUtils.DefaultString`